src/azure-cli-core/azure/cli/core/__init__.py

```diff
@@ -389,9 +389,17 @@ def _get_extension_suppressions(mod_loaders):
         self.command_group_table.clear()
         self.command_table.clear()
 
+        # Configure the commands that require all modules to be loaded
+        # Such as: "az next" need to support searching the contents of other module commands
+        full_loaded_commands = ['next']
```

Comment:
Let's put this at the beginning of this file and change it too `SKIP_COMMAND_INDEX_FOR`.

Reply:
Done